app/helpers/lessons_helper.rb

```diff
@@ -0,0 +1,8 @@
+module LessonsHelper
+  def duration_in_words(lesson)
+    duration = []
+    duration.push(pluralize(lesson.duration_hours, "hour")) if lesson.duration_hours > 0
```

Comment:
We should either make the field required or make this survive the case where duration is nil.

Reply:
Ah good point. I'll push that and also fix the linting error.

Reply:
I guess we'll probably have to require it at some point (or handle empty input somehow) once we start summing durations.